### DISCUSSION TITLE
How to debug or print our

### DISCUSSION THREAD

**Discussion by ojcloveu** [Huachao/vscode-restclient#781] [Q&A]
How to debug?

**Comment by Huachao** [comment]
@ojcloveu Do you mean debug the outgoing http request?

**Comment by dyardy** [comment]
How best to get the results of api call and output to console/window to ensure the value is correct?

```
# @result
GET {{hostname}}/start_session
content-type: application/json
Authorization: Basic {{username}}:{{password}}

@sessionId = {{result.body.$.session_Id}}
```
(how do i see the returned value from @sessionId)
